jax/_src/debugger/core.py

```diff
@@ -113,19 +114,45 @@ def register_debugger(name: str, debugger: Debugger, priority: int) -> None:
 debug_lock = threading.Lock()
 
 
-def breakpoint(*, ordered: bool = False, backend=None, **kwargs):  # pylint: disable=redefined-builtin
-  """Enters a breakpoint at a point in a program."""
+def breakpoint(*, backend: Optional[str] = None, filter_frames: bool = True,
+               use_globals: bool = False, num_frames: Optional[int] = None,
+               ordered: bool = False, **kwargs):  # pylint: disable=redefined-builtin
+  """Enters a breakpoint at a point in a program.
+
+  Args:
+    backend: The debugger backend to use. By default, picks the highest priority
+      debugger and in the absence of other registered debuggers, falls back to
+      the CLI debugger.
+    filter_frames: Whether or not to filter out JAX internal frames.
+    use_globals: Whether or not to allow inspecting global variables in the
+      debugger.
+    num_frames: The number of frames above the current to inspect in the
```

Comment:
```suggestion
    num_frames: The number of frames above the current stack frame to inspect in the
```

Reply:
Is the 'inspect' word standard? It might be the kind of word that makes sense if you already know (or can guess) what's going on, but isn't clear enough to someone with low context.

I wonder if we can try to use language mentioned in pdb's documentation. E.g. is 'inspect' used there?

An alternative to "inspect" might be "make available for inspection in the interactive debugger" or something. Just brainstorming; up to you.

Reply:
`inspect` does pop up in the `pdb` documentation but not frequently enough to justify using it. I removed use of "inspect" here.